base/sysimg.jl

```diff
@@ -180,6 +180,27 @@ big(n::Integer) = convert(BigInt,n)
 big(x::FloatingPoint) = convert(BigFloat,x)
 big(q::Rational) = big(num(q))//big(den(q))
 
+const _fact_table128 =
```

Comment:
Ok. So should they be moved to `intfuncs.jl` or `combinatorics.jl` since it already has the other variants?